easybuild/easyblocks/m/motioncor2.py

```diff
@@ -91,12 +101,16 @@ def install_step(self):
         """
 
         if (LooseVersion(self.version) >= LooseVersion("1.4")):
-            matches = glob.glob(os.path.join(self.builddir, '%s*' % self.motioncor2_bin))
+            pattern1 = os.path.join(self.builddir, '%s*' % self.motioncor2_bin)
+            pattern2 = os.path.join(self.builddir,
+                                    "{}_{}".format(self.name, self.version),
+                                    '%s*' % self.motioncor2_bin)
+            matches = glob.glob(pattern1) + glob.glob(pattern2)
             if len(matches) == 1:
                 src_mc2_bin = matches[0]
             else:
                 raise EasyBuildError(
-                    "Found multiple, or no, matching MotionCor2 binary named %s*" % self.motioncor2_bin
+                    "Found multiple, or no, matching MotionCor2 binary named %s" % pattern
```

Comment:
@stefan-wolfsheimer This is clearly wrong, since there is no variable named `pattern`....
You probably still want to use `self.motioncor2_bin` like it was before (and leave the `*` in place too)?